grp/simple.gi

```diff
@@ -860,6 +904,7 @@ local nam,e,EFactors,par,expo,prime,result,aut,i;
 	return dd;
       else
 	Error("mixed triality not yet done");
+        return false;
```

Comment:
You could also do `ErrorNoReturn` here instead of the `Error` and then `return false`. There's no problem with this way, though. (Unless I'm missing something.)